src/test/java/edu/kis/powp/command/RectangleFactory.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class RectangleFactory {
+
+    public static ComplexCommand createRectangle(DriverManager driverManager) {
```

Comment:
DriverCommand return type would be sufficient

Passing a manager instead of a driver increases dependencies.